index.html

```diff
@@ -1,506 +1,534 @@
+					<li class="active"><a href="index.html" title="Home">Home</a></li>
+					<li><a href="#aboutID" title="About us" class="page-scroll">About Us</a></li>
+					<!--
+					<li><a href="#sponsorID" title="Sponsors" class="page-scroll">Sponsors</a></li>
+					<li><a href="#speakerID" title="Speakers" class="page-scroll">Speakers</a></li>
+					-->
+					<li><a href="#socialID" title="Social Area" class="page-scroll">Social Area</a></li>
+				    </ul>
+				</div>
+			    </div>
+			    <!--
+			    <div class="reg"><a href="#" title="Register"><i class="fa fa-user" aria-hidden="true"></i>
+				    Register</a></div>
+			</div>
+			    -->
+		    </div>
+		</div>
+	    </nav>
+	</div>
+    </header>
+    <section class="banner_area">
+	<div id="clouds">
+	    <div class="cloud x1"></div>
+	    <div class="cloud x2"></div>
+	    <div class="cloud x3"></div>
+	    <div class="cloud x4"></div>
+	    <div class="cloud x5"></div>
+	    <div class="cloud x4"></div>
+	    <div class="cloud x5"></div>
+	</div>
+	<div class="container">
+	    <div class="row">
+		<div class="col-md-8 col-md-offset-2">
+		    <div class="banner_conntent">
+			<div class="ribbon_area">
+			    <h2 id="headline">GopherCon Israel</h2>
+			    <div class="ribbon" id="headline1"><i>11,</i> February, <i>2019</i></div>
+			</div>
+			<div class="inner_content">
+			    <h1>Tel Aviv Convention Center</h1>
+			    <p>GopherCon Israel 2019 is the first GopherCon to be held in Israel. </p><a href="#newsID" title="Subscribe for Updates"
+															class="btn_round page-scroll">Subscribe for Updates</a>
+			</div>
+		    </div>
+		</div>
+	    </div>
+	    <div class="bear_area">
+		<div class="row">
+		    <div class="col-md-5 col-sm-5 col-md-offset-3 col-sm-offset-3">
+			<div class="for_ball">
+			    <div class="ball_inner"><img alt="" src="images/ball.png" class="img_ball"></div>
+			    <div class="row">
+				<div class="col-md-6 col-sm-6 col-xs-3">
+				    <div class="animate_area b_bear"><img alt="" src="images/bbear.png" class="b_bear"></div>
+				</div>
+				<div class="col-md-6 col-sm-6 col-xs-3">
+				    <div class="animate_area r_bear"><img alt="" src="images/rbear.png" class="r_bear"></div>
+				</div>
+			    </div>
+			</div>
+		    </div>
+		</div>
+	    </div>
+	    <div class="duckbear"><img alt="" src="images/duckbear.png"></div>
+	</div>
+    </section>
+    <section class="about-section" id="aboutID">
+	<div class="container-fluid">
+	    <div class="row">
+		<div class="col-md-5 col-md-offset-1">
+		    <div class="about_text">
+			<h3 class="wow slideInLeft" data-wow-duration="1s">About Us</h3>
+			<h2 class="wow slideInLeft" data-wow-duration="1.3s">GopherCon is focused on<br>the Go programming
+			    language</h2>
+			<p class="wow slideInLeft" data-wow-duration="1.8s">
+			GopherCon Israel 2019 is the first GopherCon to be held in
+			Israel. The conference will be held on February 11 at the
+			Tel-Aviv convention center. 250 Gophers will attend the
```

Comment:
Tel Aviv **C**onvention **C**enter

Reply:
> There are additional links in the page that requires update
And they are?

Reply:
> Don't comment out sections, please add hidden-md and hidden-sm as class attributes (this should hide the section
Why is it better than a comment?

Reply:
> What about Google Analytics?
Will add.

Reply:
At the footer there are facebook, twitter, code of conduct etc links

Reply:
Done